index.markdown

```diff
@@ -101,6 +101,14 @@ title="ro-crate-java"
 subtitle="A library to package and link all your research data"
 content="Using ro-crate-java, you can create and modify research packages (RO-Crates). These packages allow you to describe your research machine-readable and human-readable, making it FAIR. Build this functionality into your applications or use it directly."
 %}
+
+{% include card.html tags="engineering data repositories"
```

Comment:
The currently available tags can be seen at line 117. See also line 132 and 133: in case you add tags, you will need to adjust these lines too. I propose:

- change "repositories" to the existing tag "repository"
- consider adding the tag "engineering" (maybe we should discuss this one in our HMC meeting, as we currently have more technological tags, not domain-specific ones, but I think it is fine)